subsys/mgmt/Kconfig

```diff
@@ -57,18 +56,60 @@ config MCUMGR_SMP_UART_MTU
 
 endif
 
+config MCUMGR_SMP_UDP
+	bool "UDP mcumgr SMP transport"
+	select MCUMGR
+	select NETWORKING
+	select NET_IPV4
```

Comment:
Instead of selecting IPv4 here, you could leave this empty and let the application writer to select what protocol he/she wants to use.